ComponentKit/Core/CKComponentLifecycleManager.mm

```diff
@@ -128,6 +128,11 @@ - (void)updateWithStateWithoutMounting:(const CKComponentLifecycleManagerState &
   _state = state;
 }
 
+- (void)updateContext:(id)newContext
+{
+  _context = newContext;
```

Comment:
@benlodotcom Other methods don't have this assert. I wonder should I add it here and in other place @adamjernst mentioned?